server/src/main/resources/logback-spring.xml

```diff
@@ -39,43 +39,22 @@
 
     <!-- logstash-appender: off by default -->
     <!--<include resource="logback/logstash-appender.xml"/>-->
-    <!-- metric-appender: off by default -->
```

Comment:
you can revert this file,  method name not prrint issue already fixed in: 
https://github.com/seata/seata/pull/5490/files

Reply:
done